galaxy_ng/app/models/container.py

```diff
@@ -67,6 +71,12 @@ def get_connection_fields(self):
 
         return result
 
+    def get_registry_backend(self):
+        for registry in self.RED_HAT_REGISTY_DOMAINS:
+            if self.url in registry:
+                return 'redhat'
+        return None
```

Comment:
If the idea is to match 

self.url ({https, http}://registry.redhat.io) with (registry.redhat.io) this should be compared in the opposite way,

```suggestion
        for registry in self.RED_HAT_REGISTY_DOMAINS:
            if registry in self.url:
                return 'redhat'
        return None
```